src/FormatFactory.cpp

```diff
@@ -38,7 +39,6 @@ using namespace chemfiles;
 
 namespace chemfiles {
     extern template class Molfile<DCD>;
-    extern template class Molfile<TRR>;
```

Comment:
You can also remove the corresponding value in MolfileFormat enum

Reply:
Done for both formats.